### ISSUE TITLE
run code error with "dynamically require"

### ISSUE THREAD

**Issue by kevi-sun** [asaaki/rust-wasm-on-lambda-edge#396]
## Bug Report

I mostly worked on Rust and recently need to deploy some of our services written in Rust to AWS Lambda@Edge and this is a great example to start with, thanks

the test code running well both in local and AWS cloudfront. I use this as the framework and link our code as crates, and then I have the following error when run "make call" I don't really known how to solve it (I've searched the rollup document but still don't get how to solve this, since I don't use rollup before).

do you know what it might issue with this, thanks a lot!

```
Error: Could not dynamically require "./snippets/app_edge-d7b4f29aba445e31/inline0.js". Please configure the dynamicRequireTargets or/and ignoreDynamicRequires option of @rollup/plugin-commonjs appropriately for this require call to work.
    at _ (/opt/work/app_edge/request/build.function/index.js:1:241)
    at /opt/work/app_edge/request/build.function/index.js:1:5422
    at Object.<anonymous> (/opt/work/app_edge/request/build.function/index.js:1:16865)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/opt/work/app_edge/request/fixtures/viewer-request-call-test.js:1:23)
make: *** [Makefile:58: call] Error 1
```

### Environment
Rust: 1.59.0
OS: ubuntu:20.04

**Comment by asaaki** [comment]
Hi @kevi-sun,

I could not reproduce your issue.

In the meantime I also made a whole round of updates for all parts.
Furthermore I only test on node v14 and v16, in CI with ubuntu-22.04 (using latest stable Rust, but that does not matter here).

Could you retry again? Also specific versions for node/npm/rollup CLI would maybe be needed for further debugging, though I cannot help you that much with it, I only utilize JS to the degree it helps me to achieve my goals, I'm no expert in it.

In 2022 rollup might also not be the best choice I guess, I don't really follow the trend anymore. 🤷🏻 

I nerdsniped myself and created this experiment with esbuild, which seems to be much lighter on the dependency part, but might create slightly less optimized/smaller packages: https://github.com/asaaki/rust-wasm-on-lambda-edge/pull/418

**Comment by asaaki** [comment]
So I merged #418 and the node bundling part should be less painful hopefully.

**Comment by kevi-sun** [comment]
Hi @asaaki,
I'm sorry for the late reply and thanks a lot for your keep improving this crate, your work really helped me.

The issue happen when there was the following inline js in rust that I'm not aware of. it is handy when a struct is exported from rust to js and then returned to rust (js is not really used it, just stores the value there). It is fine if move this inline function to js.

`#[wasm_bindgen(inline_js = "export function toReceiverMessage(msg) { return msg; }")]
extern "C" {
    fn toReceiverMessage(msg: JsValue) -> InternalReceiverMessage;
}`
